transport-native-io_uring/src/main/java/io/netty/channel/uring/AbstractIOUringChannel.java

```diff
@@ -247,29 +243,16 @@ protected void doClose() throws Exception {
 
             cancelConnectTimeoutFuture();
 
-            if (isRegistered()) {
-                // Need to check if we are on the EventLoop as doClose() may be triggered by the GlobalEventExecutor
-                // if SO_LINGER is used.
-                //
-                // See https://github.com/netty/netty/issues/7159
```

Comment:
This is not needed anymore as even with SO_LINGER we are fine as the close will not block this method ;)